assets/scss/_documentation.scss

```diff
@@ -19,223 +19,170 @@
-    max-width: 80%;
+  h4, h4.alert-heading {
+    color: #000;
+    display: block;
+    float: initial;
+    font-size: 1rem;
+    padding: 0;
+    padding-right: 0.5rem;
+    margin: 0;
+    line-height: 1.5; // match paragraph style
   }
 }
 
-// Special color for third party content disclaimers
-.alert.third-party-content { border-left-color: #222 };
+// Only danger headings need large text (per Docsy)
+// For other callouts, use the size of the running text
+.alert:not(.alert-danger) h4 {
+  font-size: 1em;
+}
 
-// Highlight disclaimer when targeted as a fragment
+// All callout headings should be bold
+.alert {
+  border-style: solid;
+  border-width: 0 0 0 4px;
+  border-radius: 0 0 2rem 0;
+  h4 {
+    font-weight: bold;
+    font-style: initial;
+  }
+}
 
-#third-party-content-disclaimer {
-  color: #000;
-  background: #f8f9fa;
-  transition: all 0.5s ease;
+.glossary-tooltip {
+  display: inline-block;
+  border-bottom: 1px dotted black;
+  color: $black;
+  background: transparent;
+  text-decoration: none !important;
 }
 
-@keyframes disclaimer-highlight {
-  from { background: #f8f922; color: #000; }
-  50% { background: #f8f944; color: #000; }
-  to { background: #f8f9cb; color: #000; }
+/* code samples */
+.code-sample > .copy-code-icon {
+  cursor: pointer;
+  display: flex;
+  gap: 1rem;
+  justify-content: right;
+  padding: 0.2rem;
 }
 
-#third-party-content-disclaimer:target {
-  color: #000;
-  animation: disclaimer-highlight 1.25s ease;
-  background: #f8f9cb;
+/* CSS for 'figure' full-screen display */
+
+/* Define styles for full-screen overlay */
+.figure-fullscreen-overlay {
+  position: fixed;
+  inset: 0;
+  z-index: 9999;
+  background-color: rgba(255, 255, 255, 0.95); /* White background with some transparency */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: calc(5% + 20px);
+  box-sizing: border-box;
 }
 
-.deprecation-warning, .pageinfo.deprecation-warning {
-    padding: clamp(10px, 2vmin, 20px);
-    margin: clamp(10px, 1vh, 20px) 0;
-    background-color: #faf5b6;
-    color: #000;
+/* CSS class to scale the image when zoomed */
+.figure-zoomed {
+  transform: scale(1.2);
 }
 
-.deprecation-warning.outdated-blog, .pageinfo.deprecation-warning.outdated-blog {
-    background-color: $primary;
-    color: $white;
+/* Define styles for full-screen image */
+.figure-fullscreen-img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain; /* Maintain aspect ratio and fit within the container */
 }
 
-body.td-home .deprecation-warning, body.td-blog .deprecation-warning, body.td-documentation .deprecation-warning {
-    border-radius: 3px;
+/* Define styles for close button */
+.figure-close-button {
+  position: absolute;
+  top: 1%;
+  right: 2%;
+  cursor: pointer;
+  font-size: calc(min(5vw + 10px, 4rem));
+  color: $primary;
+  background-color: rgba(255, 255, 255, 0.25);
 }
 
-.deprecation-warning p:only-child {
-  margin-bottom: 0;
+/* Sidebar menu */
```

Comment:
@sftim I was going through the sidebar code and realised that the little `(EN)` in the sidebar menu for menu items not i18nised is no longer visible. It used to be as I can see in this preview https://deploy-preview-48741--kubernetes-io-main-staging.netlify.app/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice/ from #48741 
![image](https://github.com/user-attachments/assets/ccf36749-6661-46e0-8192-8a89b92be53a)
Since the code is still there in the template but hidden from the SCSS, I was wondering if one of them is supposed to be removed.

Reply:
This PR isn't supposed to break that little detail. Do you have capacity to send in a fix PR?

Reply:
Yeah, I can send a quick fix PR or I can add this fix in the overall alignment PR that I am working on for the sidebar.